src/components/Settings/SaveSearchItem.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+	<main class>
```

Comment:
We probably don't want `main` here, since that semantic tag is meant for the main content of the document. For example, we currently use that tag in the page container components. Using `div` here would work, or a semantic tag like `article`. Also, the empty `class can be removed.